anms-ui/public/app/components/management/builder/buildAri.vue

```diff
@@ -186,8 +183,10 @@ export default {
       });
     },
     updateResults: function (result) {
-      let head = result[0].value.includes("ari") ? "" : "ari://";
-      this.finResultStr = head + result[0].value;
+      if(result[0].value){
+        let head = result[0].value.includes("ari") ? "" : "ari://";
```

Comment:
The two-slash prefix is for object and namespace references only, is that okay here? Or is this intended for literal values also?
For example the list `ari:/ac/(1,2,3)' to top-level execution like `ari:/execset/...`